plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergColumnHandle.java

```diff
@@ -133,11 +135,25 @@ public String getQualifiedName()
         return String.join(".", pathNames.build());
     }
 
+    @JsonIgnore
     public boolean isBaseColumn()
     {
         return path.isEmpty();
     }
 
+    @JsonIgnore
+    public boolean isIcebergRowPositionMetadataColumn()
+    {
+        return id == ROW_POSITION.fieldId();
+    }
+
+    @JsonIgnore
+    // Marker column used by the Iceberg DeleteFilter to indicate columns which are deleted by equality deletes
```

Comment:
`to indicate columns` -> `to indicate rows` right?

Reply:
Absolutely. Thanks for catching